src/app/pages/news/11.2/03-inscription-view.md

```diff
@@ -1,6 +1,13 @@
 ## Inscription View (Beta) {#inscriptionView}
 
-Blah inscription view blah 
+One step further on our path to finalize our new web-based Inscription Views!
+We replaced default inscriptions from the old SWT-based software and experience to the future of inscription editing.
+Beside of new features, we integrated it full inside of the process editor and give it a big UI / UX update.
+
+- __Process Editor integration__: The new inscription view is fully integrated into the existing process editor, to provide you the best user experience.
+  Implemented as a sidebar, a double click let it open and the normal selection will update the view.
```

Comment:
a double click upon an element opens it and sub-sequent selections will update the view